entries/d/datev.de.json

```
@@ -0,0 +1,15 @@
+{
+  "DATEV": {
+    "domain": "datev.de",
+    "tfa": [
+      "totp"
+    ],
+    "documentation": "https://apps.datev.de/help-center/documents/1022936",
```

There are other methods available as well:
```suggestion
      "totp",
      "sms",
      "custom-hardware"
    ],
    "custom-hardware": [
      "National identity card"
    ],
    "documentation": "https://apps.datev.de/help-center/documents/1022726",
    "notes": "2FA only available for DATEV Arbeitnehmer online",
```